mycroft/util/format.py

```diff
@@ -35,6 +37,34 @@
 import datetime
 import re
 
+
+def _translate(name, lang):
```

Comment:
We should try to use the `mycroft.dialog.get()`, it takes a bit of restructuring to get going, but see the branch [feature/new-formatters](https://github.com/MycroftAI/mycroft-core/tree/feature/new-formatters2)

Reply:
Avoided calling these files .dialog to avoid confusion.  Using .word instead.